Assets/MixedRealityToolkit.Providers/WindowsVoiceInput/WindowsSpeechInputProvider.cs

```diff
@@ -157,19 +157,24 @@ public override void Disable()
             {
                 StopRecognition();
                 keywordRecognizer.OnPhraseRecognized -= KeywordRecognizer_OnPhraseRecognized;
+
+                keywordRecognizer.Dispose();
```

Comment:
I'm not sure it makes sense to `Dispose` the keyword recognizer fully when just disabling the system.
This should also be happening already in the system's Dispose call. Are you not seeing that called reliably?

Reply:
I could do this in `Destroy` (Dispose is not enough, because I'm not getting a `Dispose` call on a service before the next test starts). I thought it makes sense here to mirror initialization in `Enable`. It will make sense if we will ever want to dynamically add remove voice keywords. Don't know if it makes sense?

---
In reply to: [287527461](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/4498#discussion_r287527461) [](ancestors = 287527461)

Reply:
This also might regress #4439. @wiwei?